notebook-images/base/jupyter-datascience-notebook-imagestream.yaml

```diff
@@ -13,6 +13,19 @@ spec:
   lookupPolicy:
     local: true
   tags:
+  # N Version of the image
+  - annotations:
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"}]'
+      openshift.io/imported-from: quay.io/opendatahub/workbench-images
+      opendatahub.io/workbench-image-recommended: 'true'
+    from:
+      kind: DockerImage
+      name: quay.io/opendatahub/workbench-images:jupyter-datascience-ubi9-python-3.9-2023a-weekly
```

Comment:
this should always be in digest format @sha256 to support airgapped and ImageContentSourcePolicy under OpenShift.
Refer to the current digest value here in tag.from.name  `quay.io/opendatahub/workbench-images@sha256:655f961295409d0d3a14927123ead8fff1b50b966948bcf18d10909e3ab32954`

Reply:
See https://github.com/opendatahub-io/odh-manifests/pull/746#issuecomment-1484430943 about using digest image hashes odh-manifests releases